### PR TITLE
Refresh example app with newest template

### PR DESCRIPTION
**Pull request by frankus**

### CODE REVIEW

**Comment by frankus** [comment]
Haven't been able to run the Android version due to MS screwing with my JDK.

**Comment by PoornimaApptentive** [comment]
> Haven't been able to run the Android version due to MS messing with my JDK.

I will take a look at it for Android. Let me merge my previous PR before that.

File: example/android/gradlew.bat
Comment (PoornimaApptentive) [inline example/android/gradlew.bat:1]: Did you make these changes or generated somehow? I have never been to these files ever, wondering how to validate

**Comment by PoornimaApptentive** [comment]
I can't make the example app run with or without these changes. But I am not sure if it is good to go with these updates without being tested. However, if it is crucial for iOS it should go.

**Review by PoornimaApptentive — APPROVED** [review]

**Comment by frankus** [comment]
Let's hold this till after the 6.7.0 release.

**Pull request closed without merging**